package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-emoji-render",
-  "version": "1.2.4",
+  "version": "1.3.1",
```

Comment:
We do not want to bump the version just yet, we will do that in a different PR towards next release 👍🏻